client-next/src/components/MapView/MapView.tsx

```diff
@@ -35,6 +35,20 @@ export function MapView({
   const onMapDoubleClick = useMapDoubleClick({ tripQueryVariables, setTripQueryVariables });
   const [showContextPopup, setShowContextPopup] = useState<LngLat | null>(null);
   const [showPropsPopup, setShowPropsPopup] = useState<PopupData | null>(null);
+  const showFeaturePropPopup = (
+    e: mapboxgl.MapMouseEvent & {
+      features?: mapboxgl.MapboxGeoJSONFeature[] | undefined;
+    },
+    setShowPropsPopup: (value: ((prevState: PopupData | null) => PopupData | null) | PopupData | null) => void,
```

Comment:
You don't need to pass this setter function since it is already in the enclosing scope

Reply:
Perhaps you meant it to extract the function outside of this scope?

Reply:
The reason I fuzz about this is that it is considered an anti-pattern to redefine / override variable names in nested scopes.

Reply:
That was some Intellij refactoring going wrong. Fixed in ace6a380f380b1e654ce63e22d1cba81d21083a8